README.md

```diff
@@ -729,6 +729,32 @@ use these keys to verify a downloaded file.
 
 </details>
 
+### Security Release Stewards
+
+When possible the commitment to take slots in the
+security release steward rotation is made by companies in order
+to ensure individuals who act as security stewards have the
+support and recognition from their employer to be able to
+prioritize security releases. Security release stewards manage security
+releases on a rotation basis as outlined in the
+[security release process](./doc/contributing/security-release-process.md).
+
+The current set of security release stewards is as follows:
+
```

Comment:
Honestly, I think this and all of the preceding explanatory text should be removed. Just list the individuals and their employers.